go/vt/vttablet/tabletserver/planbuilder/plan_test.go

```diff
@@ -268,6 +268,41 @@ func TestMessageStreamingPlan(t *testing.T) {
 	}
 }
 
+func TestLockPlan(t *testing.T) {
+	testSchema := loadSchema("schema_test.json")
+	for tcase := range iterateExecFile("lock_cases.txt") {
+		t.Run(tcase.input, func(t *testing.T) {
+			var plan *Plan
+			var err error
+			statement, err := sqlparser.Parse(tcase.input)
+			if err == nil {
+				plan, err = Build(statement, testSchema, false, "dbName")
+			}
+
+			var out string
+			if err != nil {
+				out = err.Error()
+			} else {
+				bout, err := json.Marshal(plan)
+				if err != nil {
+					t.Fatalf("Error marshalling %v: %v", plan, err)
```

Comment:
nit: If you are working on these tests, might as well update them to follow how we write tests these days - with `utils.Diff` and `require.NoError`